benchmarks/aggregate.scenario.ts

```diff
@@ -0,0 +1,25 @@
+import { MaggregorBenchmarkScenario } from 'benchmarks';
+import { MongoClient } from 'mongodb';
+
+const scenario: MaggregorBenchmarkScenario = {
+  name: 'aggregate',
+  description: 'group by city and calculate a sum | no changes',
+  // Maggregor must be at least 10x faster than MongoDB
+  expectedSpeedTreshold: 10,
```

Comment:
We should use a percentage differencial. X times faster/slower are always confusing